qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -283,6 +286,18 @@ HOCRPdfExporter::HOCRPdfExporter(const HOCRDocument* hocrdocument, const HOCRPag
 	ui.comboBoxImageCompression->addItem(_("Jpeg (lossy)"), PDFSettings::CompressJpeg);
 	ui.comboBoxImageCompression->setCurrentIndex(-1);
 
+	ui.comboBoxPaperSizeMetric->addItem(_("mm"), static_cast<int>(PaperSizeMetrics::mm));
+	ui.comboBoxPaperSizeMetric->addItem(_("inch"), static_cast<int>(PaperSizeMetrics::inch));
```

Comment:
I'd use the abbreviation for "inch", since you're also using "mm".

Reply:
What is the abbreviation for inch?

Reply:
in

Reply:
Fixed